common/djangoapps/student/management/commands/retrieve_unsubscribed_emails.py

```diff
@@ -0,0 +1,123 @@
+"""Management command to retrieve unsubscribed emails from Braze."""
+
+import logging
+import tempfile
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.core.management.base import BaseCommand, CommandError
+from django.template.loader import get_template
+
+from lms.djangoapps.utils import get_braze_client
+
+logger = logging.getLogger(__name__)
+
+UNSUBSCRIBED_EMAILS_MAX_LIMIT = 500
+
+
+class Command(BaseCommand):
+    """
+    Management command to retrieve unsubscribed emails from Braze.
+    """
+
+    help = """
+    Retrieve unsubscribed emails from Braze API based on specified parameters.
+
+    Usage:
+    python manage.py retrieve_unsubscribed_emails [--start-date START_DATE] [--end-date END_DATE]
+
+    Options:
+      --start-date START_DATE   Start date (optional)
+      --end-date END_DATE       End date (optional)
+
+    Example:
+        $ ... retrieve_unsubscribed_emails --start-date 2022-01-01 --end-date 2023-01-01
```

Comment:
The example command is incorrect
```suggestion
        $ ... retrieve_unsubscribed_emails --start_date 2022-01-01 --end_date 2023-01-01
```

Reply:
Addressed